lib/java/com/google/android/material/lists/res/layout/line_item_meta.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?><!--
+  ~ Copyright (C) 2019 The Android Open Source Project
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~      http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+-->
+<TextView xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/mtrl_list_item_meta"
+    android:layout_width="wrap_content"
+    android:layout_height="32dp"
+    android:layout_gravity="top"
+    android:layout_marginStart="28dp"
+    android:gravity="bottom"
+    android:textAppearance="@style/TextAppearance.MaterialComponents.Caption"
+    tools:text="meta" /><!-- Used layout_height="32dp" instead of the recommended 28dp to have the meta text better line
+up with the primary texts in the "two_line_item" and "three_line_item" xml layouts.
```

Comment:
nit can you move the margin to a dimen and put the comment there?

Reply:
Other things could be moved to dimens too

Reply:
A reorganization of dimensions sounds great